backend/app/context/manager.py

```diff
@@ -18,6 +18,7 @@ class Context(Model):
     snapshots_state: SnapshotsState
 
 
+@cache.memoize(timeout=60)
```

Comment:
I'm not sure about this one, because when epoch state is changing (from CURRENT to PENDING for example), we might get incorrect data and people would need to wait this one minute

Reply:
Well since this happens twice every 3 months and provided we're not finalising epochs at 0:00, but few minutes afterwards, I assume we may allow this up to 60 seconds delay. Furthermore, I can lower this to 15 seconds and still get a significant boost in performance. wdyt?

Reply:
ok, let's lower it a bit then and approve